runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaPlugin.java

```diff
@@ -338,7 +339,7 @@ public void onGameTick(GameTick event)
 	{
 		WorldPoint newLocation = client.getLocalPlayer().getWorldLocation();
 
-		if (active && notifyOnce && Instant.now().isAfter(endTime))
+		if (active && notifyOnce && endTime != null && Instant.now().isAfter(endTime))
```

Comment:
What code path yields `notifyOnce = true` and `endTime = null`? As best I can tell, `notifyOnce` is only ever set to true inside `createTimer()`, and `endTime` has its value set there also.